### PR TITLE
Backport pull request #1279: Downgraded ipywidget hack

### PR DESCRIPTION
**Pull request by minrk**
PR #1279 didn't auto-apply cleanly, so I had to do a manual patch.

Opening as a PR so @jdfreder and @parente can double-check that I didn't miss something from #1279.

### CODE REVIEW

File: notebook/static/notebook/js/main.js [hunk 1]
```diff
@@ -177,8 +171,28 @@ require([
...
+    Promise.all([
+        utils.load_extensions_from_config(config_section),
+        utils.load_extensions_from_config(common_config),
+    ])
+    .catch(function(error) {
+        console.error('Could not load nbextensions from user config files', error);
+    })
+    // BEGIN HARDCODED WIDGETS HACK
+    .then(function() {
+        if (!utils.is_loaded('widgets/extension')) {
```
Comment (SylvainCorlay) [inline notebook/static/notebook/js/main.js]: Now it is `jupyter-js-widgets/extension`. I opened the corresponding PR in master.

**Comment by jdfreder** [comment]
Thanks @minrk , this looks good
🍥 🐟

**Comment by SylvainCorlay** [comment]
@jdfreder you missed my inline remark.

**Comment by jdfreder** [comment]
No I saw it, it's okay, we can backport another.

**Pull request merged**